src/amo/css/ScreenShots.scss

```diff
@@ -1,6 +1,9 @@
 @import "~core/css/inc/mixins";
 @import "~amo/css/inc/vars";
-$screenshot-height: 200px;
+
+// See: https://github.com/mozilla/addons-frontend/issues/4887
```

Comment:
I defer to @willdurand here but IMHO I think it's a little better to have *some* description so you really don't have to dig into the issue

Reply:
Sorry I missed your previous comment about the original description. I found it too verbose and not really useful, so if we can come up with something short & accurate, I'd say let's add a comment above the `See: ...`. Thanks for raising this point @rebmullin

Reply:
How about: image-height has to be 20px smaller to ensure it doesn't overflow in the screenshots section.

Reply:
I like it, thanks @Vimal-Raghubir.

Please, don't forget the period at the end because we tend to write (syntactically) correct English sentences in our comments.